### PR TITLE
Update snippets

### LINKED ISSUE #3442: Add missing snippets
### Context & proposal
Add missing Kotlin/Java snippets
- https://getstream.io/chat/docs/android/query_members/?language=swift#pagination-and-ordering
- https://getstream.io/chat/docs/android/channel_capabilities/?language=kotlin
- https://getstream.io/chat/docs/android/disabling_channels/?language=kotlin#freeze-a-channel 
- https://getstream.io/chat/docs/android/send_message/?language=kotlin#partial-update

Make sure `docs` module contains all snippets:
- https://getstream.io/chat/docs/android/unread_channel/?language=kotlin
- Snippets from Docusaurus

### PR DESCRIPTION
**Pull request by adasiewiczr**
closes #3442 

### 🎯 Goal

Update client documentation and snippets.

### 🛠 Implementation details
1. Make sure every snippet from client documentation exists in the `docs` module
2. Add missing snippets
3. Update existing snippets

### 🧪 Testing
Just take a look at added snippets.

#### General
- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] Assigned a person / code owner group (required)
- [x] Thread with the PR link started in a respective Slack channel (#android-chat-core or #android-chat-ui) (required)
- [x] PR targets the `develop` branch
- [x] PR is linked to the GitHub issue it resolves

#### Code & documentation
- [x] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [x] Affected documentation updated (KDocs, docusaurus, tutorial)

### ☑️Reviewer Checklist
- [ ] UI Components sample runs & works
- [ ] Compose sample runs & works
- [ ] Release notes and docs clearly describe changes

### CODE REVIEW

File: docusaurus/docs/Android/02-client/06-guides/01-push-notifications/04-xiaomi.mdx [hunk 1]
```diff
@@ -51,7 +51,7 @@ dependencies {
 Then, create a `XiaomiPushDeviceGenerator` and add it to the list of generators in `NotificationConfig`, which you should pass into `ChatClient.Builder` when you initialize the SDK:
 
 ```kotlin {3-7,11}
-val notificationConfig = NotificationConfig(
+val notificationsConfig = NotificationConfig(
```
Comment (kanat) [inline docusaurus/docs/Android/02-client/06-guides/01-push-notifications/04-xiaomi.mdx:54]: Didn't fully get an idea of renaming `notificationConfig` intro the plural version while the class name is still `NotificationConfig`

File: stream-chat-android-docs/src/main/kotlin/io/getstream/chat/docs/kotlin/client/cms/Moderation.kt [hunk 2]
```diff
@@ -92,11 +92,11 @@ class Moderation(val client: ChatClient, val channelClient: ChannelClient) {
         }
 
         /**
-         * @see <a href="https://getstream.io/chat/docs/android/moderation/?language=kotlin#query-banned-users">Query banned users</a>
+         * @see <a href="https://getstream.io/chat/docs/react/moderation/?language=kotlin#query-bans-endpoint">Query bans endpoint</a>
```
Comment (kanat) [inline stream-chat-android-docs/src/main/kotlin/io/getstream/chat/docs/kotlin/client/cms/Moderation.kt:95]: Did we intentionally replace `android` with `react`?

**Review by kanat — APPROVED** [review]
Left a few comments. Other than that it's looking good to me.

File: stream-chat-android-docs/src/main/kotlin/io/getstream/chat/docs/kotlin/client/docusaurus/ChatDomainMigration.kt [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+            // ChatDomain.instance().watchChannel(cid = "messaging:sampleId", messageLimit = 30).enqueue { result ->
+            //     if (result.isSuccess) {
+            //         val channelController = result.data()
+            //     } else {
+            //         // Handle error
+            //     }
+            // }
+
+            // New approach - returns the StateFlow<ChannelState?> object and performs watchChannel request
+            val channelState: StateFlow<ChannelState?> = chatClient.watchChannelAsState(cid = "messaging:sampleId", messageLimit = 30, coroutineScope = scope)
```
Comment (MarinTolic) [inline stream-chat-android-docs/src/main/kotlin/io/getstream/chat/docs/kotlin/client/docusaurus/ChatDomainMigration.kt:127]: Maybe in the future we could format function calls with more than 2 parameters like so:

```kotlin
  val channelState: StateFlow<ChannelState?> = chatClient.watchChannelAsState(
      cid = "messaging:sampleId",
      messageLimit = 30,
      coroutineScope = scope
  )
```

It's not a big deal but I think it slightly improves readability.

File: stream-chat-android-docs/src/main/kotlin/io/getstream/chat/docs/kotlin/client/docusaurus/GettingStarted.kt [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+                super.onCreate(savedInstanceState)
+                val chatClient = ChatClient.instance() // Returns the singleton instance
+            }
+        }
+    }
+
+    fun addingTheOfflinePlugin(apiKey: String, context: Context) {
+        val offlinePluginFactory = StreamOfflinePluginFactory(
+            config = Config(
+                // Enables the background sync which is performed to sync user actions done without the Internet connection.
```
Comment (MarinTolic) [inline stream-chat-android-docs/src/main/kotlin/io/getstream/chat/docs/kotlin/client/docusaurus/GettingStarted.kt:37]:
```suggestion
                // Enables background sync which is performed to sync user actions done while offline.
```

**Review by MarinTolic — APPROVED** [review]
A fine PR. :smile:

**Review by bychkovdmitry — APPROVED** [review]

**Pull request merged**